components/brave_rewards/resources/ui/storage.ts

```diff
@@ -23,6 +23,7 @@ export const defaultState: Rewards.State = {
   contributionVideos: true,
   donationAbilityYT: true,
   donationAbilityTwitter: true,
+  donationAbilityReddit: true,
```

Comment:
where is this used?

Reply:
https://github.com/brave/brave-core/pull/2665/files#diff-5e5d79bfc427f3bdc5515271e2178172R189

Reply:
correction, this was superceded with `onInlineTipSettingChange`